src/components/Button/index.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import './index.css'
+
+
+function Button ({disabled, value, handleClickButton}){
+    
+    return(
+        <button className={disabled ? 'disabled' : 'button'} onClick={handleClickButton}>{value}</button>
```

Comment:
Why should a `button` have a `css class` named `button` ? You can instead always provide the default styles to the `button` and provide the `disabled` class only when it is `disabled`. Like this:
```suggestion
        <button className={disabled ? 'disabled' : undefined} onClick={handleClickButton}>{value}</button>
```